docs/operations/disaster_recovery_garden.md

```diff
@@ -0,0 +1,207 @@
+
+### ETCD backup
+
+The Garden cluster's ETCD is managed by `etcd-druid`, which can take care performing continuous backups. This can be configured in the `garden` resource and is strongly recommended for any setup.
+
+Reference:
+- [Backup and Restore Concept](https://github.com/gardener/gardener/blob/master/docs/concepts/backup-restore.md)
+- [Garden ETCD](https://github.com/gardener/gardener/blob/master/docs/concepts/operator.md#etcd)
+- [Example Backup Configuration](https://github.com/gardener/gardener/blob/1f9458b6eb73a8d1f489f003403e16d01bd014a9/example/operator/20-garden.yaml#L86-L93)
+
+### Garden Resource
+
+The `garden` resource should be backed up in its entirety including the `status` subresource. The `status` contains critical information like the current state of credentials rotation.
+
+### Runtime Data
+
+To ensure a successful and less-disruptive restore, the following data containing state information must be backed up from the runtime cluster. These are stored as `secrets` in the **`garden` namespace** and their names are typically suffixed by content hashes and, if triggered, a hash indicating recent rotation (e.g., `ca-f6032ea0-5e58a`).
+
+#### 1\. Encryption Keys and Configurations
+
+These exist separately for both the `kube-apiserver` and the `gardener-apiserver`. Without them , data stored in etcd cannot be decrypted, leading to data loss.
+
+* `kube-apiserver-etcd-encryption-key`
+* `kube-apiserver-etcd-encryption-configuration`
+* `gardener-apiserver-etcd-encryption-key`
+* `gardener-apiserver-etcd-encryption-configuration`
+
+#### 2\. Non-Auto-Rotated Certificate Authorities (CAs)
+
+These CAs must be preserved to prevent invalidating existing credentials:
+
+* `ca`
+* `ca-front-proxy`
+* `ca-etcd`
+* `ca-etcd-peer`
+* `ca-gardener`
+* `ca-client`
+
+#### 3\. Signing Keys
+
+* `service-account-key`
+* `gardener-apiserver-workload-identity-signing-key`
+
+**Note:** It is **not** necessary to store or restore secrets that have "bundle" in their name.
+
+### Infrastructure Credentials
+
+Typically, a `garden` resources references infrastructure credentials for a DNS provider and the etcd backup bucket. For a restore to succeed, these credentials must be valid and available in the new runtime cluster.
+
+-----
+
+## Restoration Procedure
+
+The following steps detail the restoration process.
+
+### Step 1: Backup of ETCD Backups
+
+In order to avoid data loss due to mistakes it is strongly recommended to create a backup of the etcd backup. This ensures that the restore procedure can be tried again in case of failure.
+
+### Step 2: Create a new Runtime Cluster
+
+Provision a new runtime cluster where the Garden cluster will be restored into.
+
+### Step 3: Adjust CIDRs
+
+In case the new runtime cluster has different CIDRs than the previous one, the `garden` resource must be adjusted accordingly before applying it to the new cluster.
```

Comment:
```suggestion
In case the new runtime cluster has different CIDRs than the previous one, the `Garden` resource must be adjusted accordingly before applying it to the new cluster.
```